### DISCUSSION TITLE
Auto discovery for publicIPAddresses

### DISCUSSION THREAD

**Discussion by sumant5476** [tomkerkhove/promitor#1953] [Q&A]
Hi just wanted to know what will be below parameters for publicIPAddresses in metric declaration

resourceType: ???????
resourceDiscoveryGroups:
name: ?????????

**Comment by tomkerkhove** [comment]
Hi, can you elaborate a bit what you want to achieve please?

**Reply by sumant5476** [reply]
using Auto discovery I want to fetch the metrics for Public IP address from Azure so what would be the resource type and name for it

PFB the link for the metrics which I want to fetch
https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftnetworkpublicipaddresses

**Reply by tomkerkhove** [reply]
Oh that way - We don't support public IPs as a scraper yet so we do not support resource discovery for it yet, sorry.

Feel free to open a feature request and/or we are open for contributions!
